src/fundus/publishers/us/fox_news.py

```diff
@@ -14,7 +14,7 @@
 
 class FoxNewsParser(ParserProxy):
     class V1(BaseParser):
-        _paragraph_selector = CSSSelector(".article-body > p")
+        _paragraph_selector = XPath("//div[@class='article-body'] / p[text()]")
```

Comment:
The new selector does not work on some articles: e.g. https://www.foxnews.com/politics/speaker-johnson-slams-desperate-biden-calling-gop-worse-segregationists-fundraiser and https://www.foxnews.com/world/putin-puts-west-notice-flight-nuclear-capable-bomber
It's a bit weird though, because they seem to be premium articles, but I am still able to access them, even from Edge where I don't have any blockers installed. The use a slightly different class in the div block 'article-content'